hack/docs/internal/cronjob-tutorial/generate_cronjob.go

```diff
@@ -288,7 +290,8 @@ func (sp *Sample) updateController() {
 
 	err = pluginutil.InsertCode(
 		filepath.Join(sp.ctx.Dir, "internal/controller/cronjob_controller.go"),
-		`// - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.20.4/pkg/reconcile`, skipGoCycloLint)
+		fmt.Sprintf(`// - https://pkg.go.dev/sigs.k8s.io/controller-runtime@%v/pkg/reconcile`,
```

Comment:
```suggestion
		fmt.Sprintf(`// - https://pkg.go.dev/sigs.k8s.io/controller-runtime@%s/pkg/reconcile`,
```

Since `scaffolds.ControllerRuntimeVersion` is already a string, `%s` is more appropriate than `%v` - it's clearer and more explicit :)